services/apps/alcs/src/alcs/notice-of-intent-decision/notice-of-intent-decision.module.ts

```diff
@@ -23,21 +32,30 @@ import { NoticeOfIntentModificationService } from './notice-of-intent-modificati
       NoticeOfIntentDecisionDocument,
       NoticeOfIntentModification,
       NoticeOfIntentModificationOutcomeType,
+      NoticeOfIntentDecisionComponent,
+      NoticeOfIntentDecisionComponentType,
+      NoticeOfIntentDecisionCondition,
+      NoticeOfIntentDecisionConditionType,
     ]),
     forwardRef(() => BoardModule),
     CardModule,
     DocumentModule,
     NoticeOfIntentModule,
+    NoticeOfIntentSubmissionStatusModule,
   ],
   providers: [
-    NoticeOfIntentDecisionService,
+    NoticeOfIntentDecisionV1Service,
```

Comment:
I can see that you've decided not to split the decision module into v1 and v2. The initial idea for applications to have those modules split was to separate v1 features from v2 and make it difficult for someone to mismatch methods/services. I think it would be nice to place a comment regarding the somewhere here since all features for NOI decision are in one place

Reply:
Added a comment that says to be careful with imports